### PR TITLE
Update/remove bid count flag

### PR DESCRIPTION
**Pull request by scott062**
Make sure bid count shows everywhere when setting `bid_count` false

- Search Results
- HomePage Cards 
- Compare Cards 
- Bidtracker
- Condensed Bidtracker on dashboard
- Position Manager
- Position Manager Position Details
- Position Details

### CODE REVIEW

File: src/Components/Agenda/EmployeeAgendaSearch/__snapshots__/EmployeeAgendaSearch.test.jsx.snap [hunk 1]
```diff
@@ -18,6 +18,7 @@ exports[`EmployeeAgendaSearchComponent matches snapshot when CDO 1`] = `
         "handleChangeWrapper": [Function],
         "isSubscribed": [Function],
         "notifyNestedSubs": [Function],
+        "onStateChange": [Function],
```
Comment (scott062) [inline src/Components/Agenda/EmployeeAgendaSearch/__snapshots__/EmployeeAgendaSearch.test.jsx.snap:21]: 💀

Comment (elizabeth-jimenez) [reply src/Components/Agenda/EmployeeAgendaSearch/__snapshots__/EmployeeAgendaSearch.test.jsx.snap:21]: oh nooo lol 😝

**Comment by SidelineCory24** [comment]
verified all of the above test work when `bid_count` is set to false

File: src/Components/Agenda/EmployeeAgendaSearch/__snapshots__/EmployeeAgendaSearch.test.jsx.snap (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Components/Agenda/EmployeeAgendaSearch/__snapshots__/EmployeeAgendaSearch.test.jsx.snap:21]: nah - not yet

Comment (scott062) [reply src/Components/Agenda/EmployeeAgendaSearch/__snapshots__/EmployeeAgendaSearch.test.jsx.snap:21]: i'll manually remove this

**Comment by elizabeth-jimenez** [comment]
> Make sure bid count shows everywhere when setting `bid_count` false
> 
> * Search Results
> * HomePage Cards
> * Compare Cards
> * Bidtracker
> * Condensed Bidtracker on dashboard
> * Position Manager
> * Position Manager Position Details
> * Position Details

Homepage cards?

**Comment by SidelineCory24** [comment]
> > Make sure bid count shows everywhere when setting `bid_count` false
> > 
> > * Search Results
> > * HomePage Cards
> > * Compare Cards
> > * Bidtracker
> > * Condensed Bidtracker on dashboard
> > * Position Manager
> > * Position Manager Position Details
> > * Position Details
> 
> Homepage cards?

image

**Comment by elizabeth-jimenez** [comment]
> > > Make sure bid count shows everywhere when setting `bid_count` false
> > > 
> > > * Search Results
> > > * HomePage Cards
> > > * Compare Cards
> > > * Bidtracker
> > > * Condensed Bidtracker on dashboard
> > > * Position Manager
> > > * Position Manager Position Details
> > > * Position Details
> > 
> > 
> > Homepage cards?
> 
> image

yea, i was thinking Bid List rip

**Pull request merged**